src/cirrus/lib2/logging.py

```diff
@@ -30,16 +30,15 @@
             "handlers": ["standard"],
             "level": getenv("CIRRUS_LOG_LEVEL", "DEBUG"),
         },
-        "cirrus.lib": {
+        "cirrus.lib2": {
             "handlers": ["standard"],
             "level": getenv("CIRRUS_LOG_LEVEL", "DEBUG"),
         },
     },
 }
 
 
-def configure_logging():
-    logging.config.dictConfig(config)
+logging.config.dictConfig(config)
```

Comment:
Moving this into the function so it didn't run at module import was done while testing an issue that popped up when this code was migrated from cirrus-lib. I ended up solving the problem a different way, but this change inadvertently remained. Moving the logging config to the module scope ensures logging is configured correctly by default.